src/configure.py

```diff
@@ -398,11 +404,17 @@ def load_arguments():
                          help="Enable the fourth-order Laplacian for <--model=ELBDM> (for <--wave_scheme=WAVE_FD> only).\n"
                        )
 
-    parser.add_argument( "--gramfe_gpu", type=str2bool, metavar="BOOLEAN", gamer_name="GRAMFE_ENABLE_GPU",
-                         default=False,
-                         depend={"model":"ELBDM", "wave_scheme":"WAVE_GRAMFE"},
-                         constraint={ True:{"gpu":True} },
-                         help="Enable GPU for <--wave_scheme=WAVE_GRAMFE> (using CPU by default because it is faster). Must enable <--gpu>.\n"
+    parser.add_argument( "--gramfe_scheme", type=str, metavar="TYPE", gamer_name="GRAMFE_SCHEME",
+                         default="GRAMFE_MATMUL", choices=["GRAMFE_MATMUL", "GRAMFE_FFT"],
+                         depend={"model":"ELBDM", "wave_scheme": "WAVE_GRAMFE"},
+                         constraint={ "GRAMFE_MATMUL":{"gsl":True} },
+                         help="GramFE scheme for <--wave_scheme=WAVE_GRAMFE> (GRAMFE_MATMUL: faster for PATCH_SIZE=8, GRAMFE_FFT: faster for larger patch sizes).\n"
```

Comment:
Because the cost of matrix-vector multiplication grows as N^2 for larger patches and the cost of the FFT version as N log(N). But actually, I have not checked carefully when which method is faster.